plugins/source/aws/resources/services/ecr/lifecycle_policy.go

```diff
@@ -16,10 +17,16 @@ func lifeCyclePolicy() *schema.Table {
 		Name:        "aws_ecr_repository_lifecycle_policies",
 		Description: `https://docs.aws.amazon.com/AmazonECR/latest/APIReference/API_GetLifecyclePolicy.html`,
 		Resolver:    fetchRepositoryLifecyclePolicy,
-		Transform:   transformers.TransformWithStruct(&ecr.GetLifecyclePolicyOutput{}, transformers.WithPrimaryKeys("RepositoryName", "RegistryId"), transformers.WithSkipFields("ResultMetadata")),
```

Comment:
From what I understand from the godoc below, `RegistryId` can differentiate between duplicate ARNs, so it's best to have it in the PK? (registry id isn't included in the arn)

```
	// The Amazon Web Services account ID associated with the registry that contains
	// the repository.
	RegistryId *string

	// The Amazon Resource Name (ARN) that identifies the repository. The ARN contains
	// the arn:aws:ecr namespace, followed by the region of the repository, Amazon Web
	// Services account ID of the repository owner, repository namespace, and
	// repository name. For example,
	// arn:aws:ecr:region:012345678910:repository-namespace/repository-name .
	RepositoryArn *string
```

Reply:
I've addressed this in a5931bc293dfcc3067f14a9d0a36d765ffdaf7ca

Reply:
& 7f93bf84f3d451b546ca5033d9b6d020c842eb28